doc/src/devdocs/stdio.md

```diff
@@ -2,37 +2,37 @@
 
 ## Libuv wrappers for stdio
 
-`julia.h` defines [libuv](http://docs.libuv.org) wrappers for the `stdio.h` streams:
+julia.h defines [libuv](http://docs.libuv.org) wrappers for the stdio.h streams:
```

Comment:
I sympathize but I still prefer this style. Consider this line, taken from later in the file:

> There is a hack in ios.h that makes the `ios_t.bm` field line up with the `uv_stream_t.type`

Having `ios.h` formatted as code makes it look like it could be an object, as with the nearby `ios_t.bm`. If you stick to having file names unformatted, it's clearer that it isn't an object.